OpInfos for: normal, bernoulli, multinomial

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#66358 OpInfos for: normal, bernoulli, multinomial**

Test Plan:
- run tests

Differential Revision: [D31551695](https://our.internmc.facebook.com/intern/diff/D31551695)

zou3519 committed:
OpInfos for: normal, bernoulli, multinomial

Test Plan:
- run tests

[ghstack-poisoned]

zou3519 committed:
Update on "OpInfos for: normal, bernoulli, multinomial"


Test Plan:
- run tests

[ghstack-poisoned]

mruberry reviewed (approved):
Cool! I left a few minor inline comments. I'm not sure what's up with the lint build failure, however.

Lezcano commented:
Is there anything blocking this PR? It'd be nice to have it landed to be able to test the stack in https://github.com/pytorch/pytorch/pull/69631

zou3519 commented:
@Lezcano I can rebase and land this

zou3519 commented on torch/testing/_internal/common_methods_invocations.py:
```diff
@@ -2075,6 +2075,73 @@ def get_val(dtype):
+    ]
+    return sample_inputs_normal_common(self, device, dtype, requires_grad, cases, **kwargs)
+
+def sample_inputs_bernoulli(self, device, dtype, requires_grad, **kwargs):
```
Going to file an issue for generator support

zou3519 committed:
Update on "OpInfos for: normal, bernoulli, multinomial"


Test Plan:
- run tests

Differential Revision: [D31551695](https://our.internmc.facebook.com/intern/diff/D31551695)

[ghstack-poisoned]

zou3519 commented:
@zou3519 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31551695).

zou3519 committed:
Update on "OpInfos for: normal, bernoulli, multinomial"


Test Plan:
- run tests

Differential Revision: [D31551695](https://our.internmc.facebook.com/intern/diff/D31551695)

[ghstack-poisoned]

zou3519 committed:
Update on "OpInfos for: normal, bernoulli, multinomial"


Test Plan:
- run tests

Differential Revision: [D31551695](https://our.internmc.facebook.com/intern/diff/D31551695)

[ghstack-poisoned]

zou3519 commented:
@zou3519 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31551695).
